src/SOS/Strike/strike.cpp

```diff
@@ -6888,25 +6888,34 @@ DECLARE_API(Threads)
     // We need to support minidumps for this command.
     BOOL bMiniDump = IsMiniDumpFile();
 
-    if (bMiniDump && bPrintSpecialThreads)
-    {
-        Print("Special thread information is not available in mini dumps.\n");
-    }
-
     EnableDMLHolder dmlHolder(dml);
 
     try
     {
         Status = PrintThreadsFromThreadStore(bMiniDump, bPrintLiveThreadsOnly);
-        if (!bMiniDump && bPrintSpecialThreads)
+        if (bPrintSpecialThreads)
         {
-#ifdef FEATURE_PAL
-            Print("\n-special not supported.\n");
-#else //FEATURE_PAL    
-            HRESULT Status2 = PrintSpecialThreads(); 
-            if (!SUCCEEDED(Status2))
-                Status = Status2;
-#endif //FEATURE_PAL            
+            BOOL bSupported = true;
+
+            if (bMiniDump && bPrintSpecialThreads)
+            {
+                Print("Special thread information is not available in mini dumps.\n");
+                bSupported = false;
+            }
```

Comment:
Shouldn't there be an else here so the below Print doesn't also get displayed?

Reply:
I was thinking about it too...
I skipped doing it, because I reasoned MiniDumps are only true in this sense if the dump is from Windows.
Was I wrong?
I also didn't want someone going to the effort of creating a full Linux dump only to find they weren't supported either.  So I thought it was OK to emit two messages...

Reply:
If it was an `if else`, then I guess I would reverse the tests...

Reply:
Special threads are not currently supported on Linux/OSX targets.

Reply:
Confused?  That was my point... I must have missed something.

Reply:
Doesn't matter...  I cleaned it up ...